CharacterCreation/07_Items.txt

```diff
@@ -2,164 +2,178 @@
-+ percent rolled. For severe wounds (50% damage or more), roll percents under 
+Permanent Marker ($2):
+    MarkAll(tm) permanent marking pen of one of the following colors.
+
+* Black
+* Red
+* Blue
+* Dark Green
+* Orange
+* Purple
+* Gold
+* Silver
+
+MediKit ($100):
+    For normal wounds (50% damage or less) roll percents under 50. On pass, heal 
+30+ percent rolled. For severe wounds (50% damage or more), roll percents under 
 50. On pass, heal 30. If you have Skill in Medicine, see the skill sheet for 
 separate instructions.
 
-Repair kit: $100
-For normal damage (50% or less) roll percents under 50. On pass, repair 30 + 
+Repair kit ($100):
+    For normal damage (50% or less) roll percents under 50. On pass, repair 30 + 
 percent rolled. For severe wounds (50% or more), roll percents under 50. On 
 pass, repair 30.
-If you have Skill in Repair, see the skill sheet for separate instructions.
 
-Nanite Pack: $100
-Restores 2d10 + 20 nanites.
+    If you have Skill in Repair, see the skill sheet for separate instructions.
+
+Nanite Pack ($100):
+    Restores 2d10 + 20 nanites.
 
-Security Spike: $100
-+20 to a hack check on a door with a physical access panel. DC 30 to place. 
+Security Spike ($100):
+    +20 to a hack check on a door with a physical access panel. DC 30 to place. 
 Once hacked the door can be controlled remotely.
 
-Medic Needle Pack: $60
-Contains 10 needles suitable for use with several malpractice proceedures.
+Medic Needle Pack ($60):
+    Contains 10 needles suitable for use with several malpractice proceedures.
 Not recomended for medicianal use. 
 
-Single Use door brute-forcer $120:
-Hacks through the encryption on a door (up to 75 DC) and opens it.
+Door Brute-Forcer ($120):
+    Hacks through the encryption on a door (up to 75 DC) and opens it. Single 
+use device.
 
 Zip-Cuffs ($25):
-pack of 5 nilon fiber woven standard zip-tie cuffs. Strength DC 70 or INT 80 
+    Pack of 5 nilon fiber woven standard zip-tie cuffs. Strength DC 70 or INT 80 
 to break.
 
 HD Zip-Cuffs ($300):
-Pack of 5 carbon-fiber zipe-tie cuffs. Strength DC 95 or INT 105 to Break.
+    Pack of 5 carbon-fiber zipe-tie cuffs. Strength DC 95 or INT 105 to Break.
 
 ==============================
 Explosives
 ==============================
 Breaching charges ($500):
-Will remove a door from your path. Make sure no one is within 2m of the door
+    Will remove a door from your path. Make sure no one is within 2m of the door
 in any direction or they take damage as if hit by an ECSC fragmentation 
 grenade.
 
 Wall Charge ($840):
-Reduces a 1.5x1.5m section of wall to rubble, allowing entry. Clear areas 2m 
-surrounding the charge or take 70 explosive damage.
-Wall thicknesses:
-Wood - Up to 12in.
-concrete - up to 10in
-Reinforced Concrete - up to 8 in
-Steel plate - up to 2 in
+    Reduces a 1.5x1.5m section of wall to rubble, allowing entry. Clear areas 2m 
+surrounding the charge or take 70 explosive damage. Densities of material one 
```

Comment:
I believe you are describing "thickness" below, not "Densities"?

_Side note:_ the price of a wall charge should honestly be like, $100. You can buy explosives that do 120 damage in a 3m radius for $220. 

https://github.com/bleehu/Compound_X/blob/3dbd18037679a57ad9f0ee08034888a68f8257d1/CharacterCreation/04_01_Melee_%26_Thrown_Weapons.txt#L266

Reply:
Some scope creep, but ya, I thought that was a little pricey. I'll drop the price of Door breaches too.